docs/rules/no-implicit-globals.md

```diff
@@ -0,0 +1,40 @@
+# disallow `var` and named functions in global scope (no-implicit-globals)
+
```

Comment:
Hmm, maybe only in strict mode.

https://gist.github.com/josh/ce94cb2df57759637d59